content/en/service_management/app_builder/auth.md

```diff
@@ -43,7 +49,7 @@ Connections Read
 
 ### Restrict access to a specific connection
 
-Set permissions on each connection to limit modifications or restrict their use. The granular permissions include **Viewer**, **Resolver**, and **Editor**.
+Set permissions on each connection to limit modifications or restrict their use. The granular permissions include **Viewer**, **Resolver**, and **Editor**. By default, only the author of the connection receives **Editor** access. The author can choose to grant access to additional users, roles, or teams.
```

Comment:
I think Sandra just updated this part. Maybe merge master and see if you get a merge conflict?